### ISSUE TITLE
k8s 1.22 - change CRD api version to V1

### ISSUE THREAD

**Issue by bcollard** [solo-io/gloo-portal-issues#160]
**Describe the bug**
Today, we cannot deploy Gloo Portal on k8s 1.22
We have some references left with CRDs API version `apiextensions.k8s.io/v1beta1`

```
Error: INSTALLATION FAILED: failed to install CRD crds/auth_config.yaml: unable to recognize "": no matches for kind "CustomResourceDefinition" in version "apiextensions.k8s.io/v1beta1"
```

**To Reproduce**
1. kubectl apply '...'
2. curl '....'
3. See error

**Expected behavior**

**Additional context**
Add any other context about the problem here, e.g.
- Gloo Portal version
- Kubernetes version
- Gateway technology and version (e.g. Gloo Edge Enterprise v1.5.11, Istio 1.7)

**Comment by sam-heilbron** [comment]
Portal depends on the CRDs defined by the rate-limit and ext-auth dependencies. Those have been upgraded to use v1, so this change is a dependency bump to those libraries.

**Comment by bewebi** [comment]
Resolved by solo-io/dev-portal#2148, to be released in the next 1.2 beta
